comps/finetuning/docker/Dockerfile.finetune

```diff
@@ -0,0 +1,22 @@
+# Use the same python version with ray
+FROM python:3.10.14
+
+WORKDIR /root/opea-finetune
```

Comment:
please create new user and don't use root

Reply:
it's done.